### DISCUSSION TITLE
Resizing windows relative to selected window

### DISCUSSION THREAD

**Discussion by zakissimo** [kovidgoyal/kitty#4824] [Q&A]
I currently have the following binds in my kitty.conf:

```
map kitty_mod+h remote_control resize-window -i -2 -a horizontal
map kitty_mod+l remote_control resize-window -i 2 -a horizontal
map kitty_mod+j remote_control resize-window -i 2 -a vertical
map kitty_mod+k remote_control resize-window -i -2 -a vertical
```

I see that the resizing is done relative to the first window. Is it possible to set it relative to the selected window ?

**Comment by page-down** [comment]
> I see that the resizing is done relative to the first window. ...

I can't reproduce this.
These commands are applied to the currently active window by default.

You can also use the  mappable action `resize_window`.
https://sw.kovidgoyal.net/kitty/layouts/#resizing-windows

**Reply by zakissimo** [reply]
I'm sorry I realize I got things mixed up, seeing the names of the commands made it clearer to me. 

What I meant to ask was, is there a way to make mappings move the border instead ? Meaning if I have a 2*2 grid layout and my cursor is on top left, current behavior makes it counter intuitive (ctrl+down moves the border up etc...),
can I make the command "dynamic" and relative to my position ? wider and narrower get swapped when I move from left to right and same for top and bottom ?

Sorry for getting confused

**Reply by page-down** [reply]
What you need is this feature, which has not been implemented yet.

https://github.com/kovidgoyal/kitty/issues/4239

You can try the following kitten. (Note: I think there will be problems with this kitten.)
https://github.com/chancez/dotfiles/commit/52da974ecd561afaf97de66a3f1c41ae1fe5165e
